catalog/spatial/csw/spatial-csw-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -130,6 +130,18 @@
         <argument ref="inputTransformerManager"/>
     </bean>
 
+    <bean id="ddf.catalog.endpoint.csw" class="ddf.catalog.endpoint.impl.CatalogEndpointImpl">
+        <property name="url"
+                  value="${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:${org.codice.ddf.system.port}${org.codice.ddf.system.rootContext}/csw"/>
+        <property name="urlBindingName" value="cswUrl"/>
+        <property name="id" value="registry.federation.method.csw"/>
+        <property name="name" value="CSW Endpoint"/>
+        <property name="version" value="2.0.2"/>
+        <property name="description" value="The endpoint used for csw."/>
```

Comment:
I guess I expected the description to provide a brief overview of what the service provides, or maybe how to call Get Capabilities.  Just thinking for someone look to integrate this, the current description doesn't really provide any more information than the name